README.md

```diff
@@ -318,6 +318,55 @@ OpenCode is built with a modular architecture:
 - **internal/session**: Session management
 - **internal/lsp**: Language Server Protocol integration
 
+## Custom Commands
+
+OpenCode supports custom commands that can be created by users to quickly send predefined prompts to the AI assistant.
+
+
+### Creating Custom Commands
+
+Custom commands are just predefined prompts stored as Markdown files in the `commands` directory within your OpenCode data directory:
+
+Global 
+```
+$HOME/.opencode/commands/
```

Comment:
I don't think the code is actually looking for this global dir? maybe I am missing something but I think this is a good idea to add.

Reply:
Fixed